meson.build

```diff
@@ -114,9 +115,12 @@ libportal_dep = dependency('libportal',
 pipewire_dep = dependency('libpipewire-0.3', version: '>= 0.2.90')
 libsystemd_dep = dependency('libsystemd', required: get_option('systemd'))
 
-
 bwrap = find_program('bwrap', required: get_option('sandboxed-image-validation'))
 
+if not bwrap.found()
+  bwrap = find_program('bwrap', required: get_option('sandboxed-sound-validation'))
+endif
```

Comment:
This code makes no sense. If bwrap was found you don't need to search for it again. You want bwrap to be required for `sandboxed-image-validation` and `sandboxed-sound-validation`. `required` is a boolean so it should be possible to set `required: get_option('sandboxed-image-validation') or get_option('sandboxed-sound-validation')`

Reply:
The options are both features so they are tristate auto, enabled, disabled. on auto and disabled, not finding bwrap is okay, if enabled, bwrap is required. This needs to be

`bwrap = find_program('bwrap', required: get_option('sandboxed-image-validation').enabled() or get_option('sandboxed-sound-validation').enabled())`

We also don't handle disabled properly. If bwrap is found and `feature.disabled()` is True, we still build everything with the bwrap sandbox.

```diff
index 433f1fc..1465bad 100644
--- a/meson.build
+++ b/meson.build
@@ -113,10 +113,14 @@ libportal_dep = dependency('libportal',
 pipewire_dep = dependency('libpipewire-0.3', version: '>= 0.2.90')
 libsystemd_dep = dependency('libsystemd', required: get_option('systemd'))
 
+bwrap = find_program('bwrap')
+if get_option('sandboxed-image-validation').enabled() and not bwrap.found()
+       error('bwrap is required because sandboxed-image-validation is enabled')
+endif
 
-bwrap = find_program('bwrap', required: get_option('sandboxed-image-validation'))
+sandboxed_image_validation = get_option('sandboxed-image-validation').allowed() and bwrap.found()
 
-if not bwrap.found()
+if not sandboxed_image_validation
   warning('''
     Sandboxed image validation with Bubblewrap is DISABLED.
     If your system can run Bubblewrap, it's **hightly** recommended that you enable this
@@ -203,7 +207,7 @@ summary({
     'Enable python test suite': enable_pytest,
     'Build man pages': rst2man.found(),
     'Build flatpak interfaces': flatpak_intf_dir != '',
-    'Sandboxed image validation': bwrap.found(),
+    'Sandboxed image validation': sandboxed_image_validation,
   },
   section: 'Optional builds',
   bool_yn: true,
diff --git a/src/meson.build b/src/meson.build
index 41f025e..0fa7f64 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -163,7 +163,7 @@ configure_file(
 )
 
 validate_icon_c_args = ['-D_GNU_SOURCE=1']
-if bwrap.found()
+if sandboxed_image_validation
   validate_icon_c_args += '-DHELPER="@0@"'.format(bwrap.full_path())
 endif

```